masonry_core/src/lib.rs

```diff
@@ -1,10 +1,42 @@
 // Copyright 2025 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
-//! Traits and types of the Masonry toolkit.
-//! See [Masonry's documentation] for more details, examples and resources.
+//! Masonry Core provides the base GUI engine for Masonry.
+//!
+//! Masonry's widgets are implemented in the Masonry crate, which re-exports this crate as `masonry::core`.
+//! Most users who wish to use Masonry for creating applications (and UI libraries) should prefer that crate.
```

Comment:
There is a level of ambiguity by saying "that crate", since the prior sentence talks about two crates. It'd probably be better to be explicit with the crate there.